types/creditcards/creditcards-tests.ts

```diff
@@ -0,0 +1,13 @@
+import { creditcards } from 'creditcards';
```

Comment:
Have you tested this exact syntax in your own code? It does not match the underlying implementation, so this most likely does not work at runtime.